.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Hardhat + EDR",
+  "image": "mcr.microsoft.com/devcontainers/base:bullseye",
+  "features": {
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": "16" /* Keep in sync with the oldest version of Node.js that Hardhat supports */
```

Comment:
I added an item for this in the "Adding and dropping support for node.js versions" checklist